### PR TITLE
Add chevron with probabilities

### LINKED ISSUE #767: Chevron with probabilities

### PR DESCRIPTION
**Pull request by andrea-pasquale**
Closes #767 
This PR adds the possibility to run chevron by measuring probabilities instead of "signal" (which is usually what I see in papers). 
For now I am measuring the probability of $\ket{1}$ for both qubits. 

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [x] Tests are passing.
- [ ] Coverage does not decrease.
- [x] Documentation is updated.
- [x] Compatibility with Qibo modules (Please edit this section if the current pull request is not compatible with the following branches).
    - [x] Qibo: `master`
    - [x] Qibolab: `main`
    - [x] Qibolab_platforms_qrc: `main`

### CODE REVIEW

**Comment by andrea-pasquale** [comment]
I just ran on hardware, the results seems to be consistent between [signal](http://login.qrccluster.com:9000/BFekyaQnRoisw1QjIyfhHQ==/) and [probability](http://login.qrccluster.com:9000/5Z12o31ZRWmJ8X18YGCifQ==/).
I would like to update the plot of the probability version to show that I am measuring the probability of measuring  $\ket{1}$.

**Review by Edoardo-Pedicillo — APPROVED** [review]
LGTM

File: src/qibocal/protocols/characterization/two_qubit_interaction/chevron/chevron_signal.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    params: ChevronSignalParameters,
+    platform: Platform,
+    targets: list[QubitPairId],
+) -> ChevronSignalData:
+    r"""
+    Perform an iSWAP/CZ experiment between pairs of qubits by changing its frequency.
+
+    Args:
+        platform: Platform to use.
+        params: Experiment parameters.
```
Comment (GabrielePalazzo) [inline src/qibocal/protocols/characterization/two_qubit_interaction/chevron/chevron_signal.py:85]:
```suggestion
        params: Experiment parameters.
        platform: Platform to use.
```

**Review by GabrielePalazzo — CHANGES_REQUESTED** [review]

**Review by igres26 — APPROVED** [review]
Looks good. After the small changes that are requested it can be merged.

**Review by GabrielePalazzo — APPROVED** [review]

**Pull request merged**